lib/OrdinaryDiffEqFIRK/Project.toml

```diff
@@ -6,15 +6,20 @@ version = "1.1.0"
 [deps]
 DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
 FastBroadcast = "7034ab61-46d4-4ed7-9d0f-46aef9175898"
+GenericLinearAlgebra = "14197337-ba66-59df-a3e3-ca00e7dcff7a"
```

Comment:
This is a pretty big set of dependencies, are we sure we want to include them by default? OrdinaryDiffEqFIRKDeriver or something that can be optionally added?